### PR TITLE
Update analytics portal url to choreo insights url

### PR DESCRIPTION
**Pull request by ruwiniwj**
## Purpose
Update analytics portal url to choreo insights url

## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: en/docs/api-analytics/getting-started-guide.md [hunk 1]
```diff
@@ -89,7 +89,7 @@ If your system connects to the service through a proxy server/firewall, you need
 
 ## Step 4 - View the Analytics Dashboards
 
-Invoke APIs and sign in to the Analytics Portal [https://analytics.choreo.dev/](https://analytics.choreo.dev/) to view the dashboards.
+Invoke APIs and open Choreo Insights [https://console.choreo.dev/insights](https://console.choreo.dev/insights) to view the dashboards.
 
 For more detailed information on the Analytics Portal dashboards and their usage, see the [Analytics Portal Overview]({{base_path}}/api-analytics/viewing/analytics-pages-overview).
```
Comment (fazlan-nazeem) [inline en/docs/api-analytics/getting-started-guide.md:94]: Shall we rename "Analytics Portal" to "Choreo Insights" as there is no more a portal for analytics?

File: en/docs/api-analytics/getting-started-guide.md [hunk 2]
```diff
@@ -89,7 +89,7 @@ If your system connects to the service through a proxy server/firewall, you need
 
 ## Step 4 - View the Analytics Dashboards
 
-Invoke APIs and sign in to the Analytics Portal [https://analytics.choreo.dev/](https://analytics.choreo.dev/) to view the dashboards.
+Invoke APIs and open Choreo Insights [https://console.choreo.dev/insights](https://console.choreo.dev/insights) to view the dashboards.
```
Comment (samgnaniah) [inline en/docs/api-analytics/getting-started-guide.md:92]: Let's make "Choreo Insights" as the link text.

File: en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/configure-analytics.md [hunk 3]
```diff
@@ -65,7 +65,7 @@ Let's generate some traffic to see the Analytics in Choreo cloud.
 
 1. Deploy your API - Follow [this]({{base_path}}/deploy-and-publish/deploy-on-gateway/choreo-connect/deploy-api/deploy-api-to-choreo-connect/) according to your setup.
 2. Let's Invoke the API few times - [Invoke the API]({{base_path}}/deploy-and-publish/deploy-on-gateway/choreo-connect/deploy-api/deploy-api-to-choreo-connect/#invoke-the-api)
-3. Go to [https://analytics.choreo.dev/overview](https://analytics.choreo.dev/overview) to view statistics. 
+3. Go to [https://console.choreo.dev/insights](https://console.choreo.dev/insights) to view statistics.
```
Comment (samgnaniah) [inline en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/configure-analytics.md:68]: Let's make the link text as Choreo Insights. Please apply everywhere.

File: en/docs/api-analytics/getting-started-guide.md (hunk 1 above)
Comment (ruwiniwj) [reply en/docs/api-analytics/getting-started-guide.md]: Fixed with eeb2775a30e95680c642a646120456cddef80314. Renamed all occurrences of  "Analytics Portal" to "Choreo Insights"

File: en/docs/api-analytics/getting-started-guide.md (hunk 2 above)
Comment (ruwiniwj) [reply en/docs/api-analytics/getting-started-guide.md]: Fixed with eeb2775a30e95680c642a646120456cddef80314.

File: en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/configure-analytics.md (hunk 3 above)
Comment (ruwiniwj) [reply en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/configure-analytics.md]: Fixed with eeb2775a30e95680c642a646120456cddef80314.

**Review by fazlan-nazeem — APPROVED** [review]

**Pull request merged**